python/tank/deploy/tank_commands/setup_project_params.py

```diff
@@ -1194,7 +1197,32 @@ def get_uri(self):
         :returns: string
         """
         return self._config_uri
-        
+
+    def is_local_configuration(self):
```

Comment:
I think local is a better them than existing_project. The example of the zipped config inside Shotgun doesn't work because this piece of code is trying to match a pipeline configuration on disk. With the zip config stuff it could be the case that the pipeline configuration is not at a specific location anymore.

Maybe configured_project?